docs/tools/hardhat/hardhat-zksync-vyper-verify.md

```diff
@@ -0,0 +1,154 @@
+# `hardhat-zksync-vyper-verify`
+
+This plugin is used to verify vyper contracts on the zkSync Era network.
+
+[Changelog](https://github.com/matter-labs/hardhat-zksync/blob/main/packages/hardhat-zksync-vyper-verify/CHANGELOG.md)
```

Comment:
This currently returns a 404. Will we have the Changelog with the relelase?

Reply:
Yes, that is correct.